deploy/azuredeploy.json

```diff
@@ -61,7 +61,12 @@
     "postCallSurveyOneQuestionPollPrompt": "",
     "postCallSurveyOneQuestionPollType": "",
     "postCallSurveyOneQuestionPollAnswerPlaceholder": "",
-    "postCallSurveyOneQuestionPollSaveButtonText": ""
+    "postCallSurveyOneQuestionPollSaveButtonText": "",
+    "cosmosDbAccountName": "[concat('db-account-', parameters('appServiceName'))]",
+    "roleDefinitionName": "[concat('sql-role-definition-', parameters('appServiceName'))]",
+    "cosmosDbName": "",
```

Comment:
We should provide a default value here since it is not a parameter/ it can easily be missed. I was able to provision the resources successfully but realized afterwards in the AppService's config that the db name was empty. I feel like that will be a common mistake.